addons/web_tour/static/src/widgets/tour_start.js

```diff
@@ -0,0 +1,49 @@
+import { charField, CharField } from "@web/views/fields/char/char_field";
+import { useService } from "@web/core/utils/hooks";
+import { registry } from "@web/core/registry";
+
+export class TourStartWidget extends CharField {
+    static template = "web_tour.TourStartWidget";
+    static props = {
+        ...CharField.props,
+        icon: { type: Boolean, optional: true },
+    };
+
+    setup() {
+        const { data } = this.props.record;
+        this.tour = useService("tour_service");
+        this.tourData = data;
+    }
+
+    _onStartTour(ev) {
+        ev.preventDefault();
```

Comment:
we don't usually prevent or/and stop events, except if there is a good reason.
or if we do (for a good reason), we usually use owl's event handler modifiers: `t-on-click.prevent="myFunc"`